appimage/BUILD

```diff
@@ -1,7 +1,62 @@
 load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+load("toolchain.bzl", "appimage_toolchain")
 
 package(default_visibility = ["//visibility:public"])
 
+toolchain_type(name = "appimage_toolchain_type")
+
+# Toolchain for linux, x86_64
+appimage_toolchain(
+    name = "appimage_linux_x86_64",
```

Comment:
I think it would be nice to generate these using a listcomp. Maybe rename the _SHAS constant from deps.bzl and iterate over RUNTIME_SHAS.keys()?

Reply:
Updated, things got a little out of hand due to the runtime arch being named slightly differently (`armhf` vs `armv7-e`)